pkg/cli/verifier/envoy_config.go

```diff
@@ -248,3 +257,120 @@ func getFilterForProtocol(protocol string) string {
 		return ""
 	}
 }
+
+func (v *EnvoyConfigVerifier) verifyDestination() Result {
+	result := Result{
+		Context: fmt.Sprintf("Verify Envoy config on destination for traffic [%s]", v.configAttr),
+	}
+
+	config, err := v.configAttr.dstConfigGetter.Get()
+	if err != nil || config == nil {
+		result.Status = Unknown
+		result.Reason = fmt.Sprintf("Error retrieving Envoy config for pod %q, err: %q", v.configAttr.trafficAttr.SrcPod, err)
+		return result
+	}
+
+	//
+	// Verify the config
```

Comment:
Did you mean to have this floating comment? I'm not sure if it is meant to be on line 265 instead.

Reply:
Yes, I was imagining a scenario like this:
```
func (v *EnvoyConfigVerifier) verifyDestination() Result {
        ...
	//
	// Verify the config
        
        // check inbound listener
        ...

        //
        // Verify something else not related to envoy config
        // e.g. selectors on k8s service
        
        // check k8s service
        ...
```

I used the floating comment to categorize a collection of code that includes comments. It's a very loose convention with the styling
```
    //
    // comment pertaining to a collection where new lines may occur
```